Makefile

```diff
@@ -128,6 +141,8 @@ cross-test:
 copy: build
 	scp target/aarch64-unknown-linux-gnu/release/piggui $(PI_USER)@$(PI_TARGET):~/
 	scp target/aarch64-unknown-linux-gnu/release/piglet $(PI_USER)@$(PI_TARGET):~/
+	scp target/armv7-unknown-linux-gnueabihf/release/piggui $(PI_USER)@$(PI_TARGET):~/
```

Comment:
I would add a separate target for this... copy-arm7 or something.

If the binary is not build for the "other" of the two architectures, the copy and the target will fail....